src/Rector/Core/Utility/RefactorDeprecationLogRector.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\Core\Utility;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Expr\StaticCall;
+use PhpParser\Node\Name;
+use PhpParser\Node\Scalar\String_;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
```

Comment:
Could you add the @see docblock to link to the actual .rst file. 

I added a phpstan rule for it but didn´t check it during integration. This is a todo for me.